.github/workflows/test-package-build.yml

```diff
@@ -9,6 +9,9 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        type: ["full", "skinny"]
```

Comment:
NIt question: How should we call the non-skinnly client 😅?

Reply:
full is great! nice naming!